app/scripts/modules/titus/src/domain/ITitusServerGroup.ts

```diff
@@ -16,6 +16,7 @@ export interface ITitusServerGroup extends IServerGroup {
   capacityGroup?: string;
   disruptionBudget?: IJobDisruptionBudget;
   entryPoint: string;
```

Comment:
looking at this now, I wonder if entryPoint should be optional too?  Are entryPoint and cmd mutually exclusive?

Reply:
agree , updated `entryPoint` to be optional , from what i see they are not mutually exclusive, titus api applies both of them if they are present.